packages/ui-extensions/src/surfaces/admin/components/CustomerSegmentTemplate/CustomerSegmentTemplate.doc.ts

```diff
@@ -0,0 +1,39 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'CustomerSegmentTemplate',
+  description:
+    'CustomerSegmentTemplate is used to configure a template used in the customers section in the admin.',
```

Comment:
consider adding a use case - when would someone want to use this template? how does segmentation factor in here (this just seems like it can be used for anything you want to append to the customers section)?

```suggestion
    'CustomerSegmentTemplate is used to configure a template used in the **Customers** section of the Shopify admin.',
```